CHANGELOG.md

```diff
@@ -21,4 +21,9 @@ All notable changes to this project will be documented in this file.
   We expect that outer edge handler to log the exception chain (the wrapper plus nested `cause` exception(s))
   and exit the process with a non-0 status code.
 
+## [1.1.1] - 2020-05-03
+- Replace hobo_support with invoca_utils
+
+[1.3.0]: https://github.com/Invoca/exceptional_synchrony/compare/v1.2.0...v1.3.0
 [1.2.0]: https://github.com/Invoca/exceptional_synchrony/compare/v1.1.1...v1.2.0
+[1.1.1]: https://github.com/Invoca/exceptional_synchrony/compare/v1.1.0...v1.1.1
```

Comment:
I still wonder if this table down here is worth it, given that it's not DRY leading to being wrong fairly often. To my knowledge, no one has ever noticed...which tells me these wouldn't be missed.

Reply:
We could automate this step when we get automated releases from CI working? I agree it's error prone and often mistaken or forgotten. Would be interesting to poll the teams on whether those links get used, or if we could just drop entirely

Reply:
I like the idea of polling the dev team. "How often have you clicked on the links in the CHANGELOG?" I know my answer would be "never" and I'm guessing most others' would be too.